libraries/standard/http/httpFilePaths.cmake

```diff
@@ -12,7 +12,8 @@ set( HTTP_SOURCES
 # HTTP library Public Include directories.
 set( HTTP_INCLUDE_PUBLIC_DIRS
      ${MODULES_DIR}/standard/http/include
-     ${MODULES_DIR}/standard/utilities/include )
+     ${MODULES_DIR}/standard/utilities/include
```

Comment:
is MODULES_DIR the /libraries folder? Because there doesn't appear to be a utilities folder there